pkgs/top-level/all-packages.nix

```diff
@@ -16120,6 +16120,8 @@ with pkgs;
 
   chromiumDev = lowPrio (chromium.override { channel = "dev"; });
 
+  chromiumUngoogled = lowPrio (chromium.override { ungoogled = true; });
```

Comment:
I don't think we should let hydra build yet another chromium. The build takes quiet long.

Reply:
OK, I haven't tested building it though, which will take a very long time.

Reply:
You should before we merge a feature like this.

Reply:
How do you intend to keep this up-to-date?

Reply:
What if I add `chromiumUnstable` back to all-packages.nix, let Borg build it once (or if Borg could build with an override), then remove it. Since the patches are fetched instead of vendored/hardcoded, to keep them up-to-date I just have to update the git rev and sha256.

Reply:
ofborg has a time limit of one hour. Chromium takes 10 hours or more.

Reply:
OK I will bite the bullet and compile chromium + ungoogled.

Reply:
Another option to prevent massive 10 hour build is to take the binary from https://github.com/Eloston/ungoogled-chromium#software-repositories (AUR) and do patchELF.

Reply:
That might be an option.

Reply:
> Another option to prevent massive 10 hour build is to take the binary from https://github.com/Eloston/ungoogled-chromium#software-repositories (AUR) and do patchELF.

I feel like I saw @srhb discuss this somewhere (apologies in advance if not)